api/src/main/java/org/apache/cloudstack/api/ApiConstants.java

```diff
@@ -821,7 +821,7 @@
     public static final String SEMANTIC_VERSION = "semanticversion";
     public static final String KUBERNETES_VERSION_ID = "kubernetesversionid";
     public static final String KUBERNETES_VERSION_NAME = "kubernetesversionname";
-    public static final String MASTER_NODES = "masternodes";
+    public static final String PRIMARY_NODES = "primarynodes";
```

Comment:
@sureshanaparti Please use Leader instead of primary to follow the kubernetes paradigm

Reply:
> @sureshanaparti Please use Leader instead of primary to follow the kubernetes paradigm

@davidjumani control nodes seems to be appropriate, is used for the kubernetes master nodes.